cmd/fluxd/main.go

```diff
@@ -660,7 +659,7 @@ func main() {
 		"sync-tag", *gitSyncTag,
 		"state", *syncState,
 		"readonly", *gitReadonly,
-		"registry-scanning", *registryScanning,
+		"registry-disable-scanning", *registryDisableScanning,
```

Comment:
I don't think this line is was really necessary in the first place (all the other things are about the git repo and syncing)

Reply:
Uhm, I think it's useful to debug problems with Flux (as important as the readonly mode).

Flux's behavior changes quite a bit.

Reply:
If it's not reporting whether scanning is disabled anywhere else in the log, I guess you could put it here. It's a bit random, though.

Reply:
Or coming from the other direction -- this line has already diverged from its original purpose, which was to report the git-specific parameters, so why not :-)